app/views/nsm/claim_details/show.html.erb

```diff
@@ -12,6 +12,11 @@
         <p><strong><%= t(".sent_back", date: claim_summary.sent_back_on.to_fs(:stamp)) %></strong></p>
         <%= simple_format claim_summary.assessment_comment %>
       </div>
+    <% elsif claim.provider_updated? && FeatureFlags.nsm_rfi_loop.enabled? %>
```

Comment:
I think this feature flag check is redundant - a claim can't get to being `provider_updated` unless the RFI loop feature was enabled when it was sent back, and if it was enabled then, we don't want to break the UX for that claim if we subsequently switch the feature flag off.